lib/std/os.zig

```diff
@@ -3169,3 +3169,13 @@ pub fn dn_expand(
     }
     return error.InvalidDnsPacket;
 }
+
+pub fn yield() void {
```

Comment:
Functions in this file are supposed to be named after POSIX function names, so this should be `sched_yield`.